bot/cogs/error_handler.py

```diff
@@ -89,6 +95,28 @@ def __init__(self, bot: Bot):
         else:
             return self.bot.get_command("help")
 
+    async def try_silence(self, ctx: Context) -> bool:
+        """
+        Attempt to invoke the silence or unsilence command if invoke with matches a pattern.
+
+        Respecting the checks if:
+                                  invoked with `shh+` silence channel for amount of h's*2 with max of 15.
+                                  invoked with `unshh+` unsilence channel
+        Return bool depending on success of command.
+        """
+        command = ctx.invoked_with.lower()
+        silence_command = self.bot.get_command("silence")
+        if not await silence_command.can_run(ctx):
```

Comment:
`can_run` may raise `CommandError`, which should be handled here. Refer to how `try_get_tag` did it.

Reply:
How do we want to handle it here?
Invoking the error handler (and returning `False`) will cause it to be triggered twice because of the `try_get_tag` that follows.
The errors could be silenced for the user with just being logged, or None can be returned to also cancel the tag fallback

Reply:
The `invoked_from_error_handler` attribute exists to prevent long or infinite recursion in the error handler when it's a `CommandNotFound`.

Reply:
Copying that behaviour (and not changing anything else), 
the `await self.on_command_error(ctx, error)` from the methods can get executed twice. Once in the silence attempt and then in the tag attempt.
`invoked_from_error_handler` doesn't come into play here because we're skipping the error handler with a `CommandError` with which it gets invoked.

Reply:
Yes, the command handler can be triggered twice if both commands error out, and that's fine. That's the point: why write code (logging, sending a message, etc.) to handle errors when the error handler already has code which can handle them?

Reply:
Do we want it to be triggered twice? It can result in duplicate messages being sent on an invalid command.

Reply:
No, probably not. Let's just go with what you said and log it directly.